src/jvmTest/java/it/krzeminski/snakeyaml/engine/kmp/usecases/references/DereferenceAliasesTest.kt

```diff
@@ -0,0 +1,47 @@
+package it.krzeminski.snakeyaml.engine.kmp.usecases.references
+
+import it.krzeminski.snakeyaml.engine.kmp.api.Dump
+import it.krzeminski.snakeyaml.engine.kmp.api.DumpSettings
+import it.krzeminski.snakeyaml.engine.kmp.api.Load
+import it.krzeminski.snakeyaml.engine.kmp.api.LoadSettings
+import it.krzeminski.snakeyaml.engine.kmp.common.FlowStyle
+import it.krzeminski.snakeyaml.engine.kmp.exceptions.YamlEngineException
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.fail
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+import org.snakeyaml.engine.v2.utils.TestUtils
+import java.io.StringWriter
+
+@Tag("fast")
+class DereferenceAliasesTest {
```

Comment:
suggestion: maybe we can move the test to common sources? Instead of using resources we could use string fields in the test class

Reply:
Sure, we should generally commonize our tests, so adding new ones in common sources is a good idea. Regarding the test resources, WDYT about using https://github.com/goncalossilva/kotlinx-resources/? I haven't used it yet, but it looks promising. I'd like to keep the test files separate for readability.

Reply:
This lib doesn't support wasmJs target as I can see... so we won't be able to use it in common sources

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/272/commits/39ea21bef4a9996cf3c84446b5b50861ff9e7bd1

Reply:
I actually thought about implementing multiplatform test resource support somewhat like @aSemy proposed here: https://github.com/goncalossilva/kotlinx-resources/issues/65#issuecomment-1587945279, by providing an implementation of kotlinx.io/Okio's FileSystem. I'll think about tackling it one day!

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/312